confgenerator/files.go

```diff
@@ -43,9 +43,6 @@ func ReadUnifiedConfigFromFile(path, platform string) (*UnifiedConfig, error) {
 	if err != nil {
 		return nil, err
 	}
-	if err := uc.Validate(); err != nil {
```

Comment:
Do we not need to call this after `confgenerator.MergeConfFiles` now?

Reply:
We run `uc.Validate` in `confgenerator.MergeConfFiles` already before returning.

Reply:
Never mind, it's already called by `confgenerator.MergeConfFiles`.